content/python/concepts/math-module/terms/math-atan/math-atan.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'math.atan()'
+Description: 'Returns the inverse tangent of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`math.atan()`** returns the arctangent of a number. If the tangent of a number x gives you n, then the arctangent of n gives you the number x. The number n can be is all real numbers. You can think of n as the slope of the line (opposite over adjacent or y/x) and x to be the degrees.
```

Comment:
```suggestion
The **`math.atan()`** returns the arctangent of a number. If the tangent of a number x gives you n, then the arctangent of n gives you the number x. The number n can be all real numbers. You can think of n as the slope of the line (opposite over adjacent or y/x) and x to be the degrees. 
```
The result of `math.atan( )` is in radians and not in degrees, so please make this edit.